### ISSUE TITLE
css for footer / header

### ISSUE THREAD

**Issue by kilianc** [marcbachmann/node-html-pdf#94]
how? it appears that only inline works

**Comment by marcbachmann** [comment]
Yes, that's right. Wrap the whole header/footer in a div and style it that way. 
You can also use html to define headers and footers in the pdf: https://github.com/marcbachmann/node-html-pdf#footers-and-headers

**Comment by floriantraber** [comment]
@marcbachmann do you mean like this: 

```
<div class="pageFooter">
    <div id="pageFooter">
```
this would wrap the footer in a div,  but I'm guessing thats wrong.

**Comment by marcbachmann** [comment]
The div with the class is useless. That will stay in the html.
By now footers & headers are also supported in the config object.

https://github.com/marcbachmann/node-html-pdf#options

I guess we can close this issue.
